connect-inject/handler_test.go

```diff
@@ -348,7 +384,23 @@ func TestHandlerHandle(t *testing.T) {
 				},
 				{
 					Operation: "add",
-					Path:      "/metadata/annotations",
+					Path:      "/metadata/annotations/" + escapeJSONPointer(annotationStatus),
+				},
+				{
+					Operation: "add",
+					Path:      "/metadata/annotations/" + escapeJSONPointer(annotationService),
+				},
+				{
+					Operation: "add",
+					Path:      "/metadata/annotations/" + escapeJSONPointer("prometheus.io/scrape"),
+				},
+				{
+					Operation: "add",
+					Path:      "/metadata/annotations/" + escapeJSONPointer("prometheus.io/port"),
+				},
+				{
+					Operation: "add",
+					Path:      "/metadata/annotations/" + escapeJSONPointer("prometheus.io/path"),
```

Comment:
If one were to make a request to add `"prometheus.io/*"` to the constants in the `annotations.go` file, how much would you hate them? Asking for a friend 😛
The friend has observed that they have been used twice in tests and once in handler.go

Reply:
Done!